lifenet/models.py

```diff
@@ -0,0 +1,90 @@
+import uuid
+
+from django.db import models
+from django.utils import timezone
+from django_prometheus.models import ExportModelOperationsMixin
+
+from healthcheck.utils import hash_string
+
+
+class LNCheck(ExportModelOperationsMixin("ln-check"), models.Model):
+    AGE_U18 = "<18"
+    AGE_18T39 = "18-39"
+    AGE_40T65 = "40-65"
+    AGE_O65 = ">65"
+    AGE_CHOICES = (
+        (AGE_U18, AGE_U18),
+        (AGE_18T39, AGE_18T39),
+        (AGE_40T65, AGE_40T65),
+        (AGE_O65, AGE_O65),
+    )
+
+    EXPOSURE_YES = "yes"
+    EXPOSURE_NO = "no"
+    EXPOSURE_NOT_SURE = "not_sure"
+    EXPOSURE_CHOICES = (
+        (EXPOSURE_YES, "Yes"),
+        (EXPOSURE_NO, "No"),
+        (EXPOSURE_NOT_SURE, "Not sure"),
+    )
+
+    RISK_LOW = "low"
+    RISK_MODERATE = "moderate"
+    RISK_HIGH = "high"
+    RISK_CHOICES = (
+        (RISK_LOW, "Low"),
+        (RISK_MODERATE, "Moderate"),
+        (RISK_HIGH, "High"),
+    )
+
+    LANGUAGE_ENGLISH = "eng"
+    LANGUAGE_FRENCH = "fr"
+
+    LANGUAGE_CHOICES = (
+        (LANGUAGE_ENGLISH, "English"),
+        (LANGUAGE_FRENCH, "Français"),
+    )
```

Comment:
Not critical, but there's a new django thing that makes this a bit cleaner: https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types